python/sqlflow_submitter/tensorflow/analyze.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import matplotlib
+import seaborn as sns
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+from sqlflow_submitter.db import connect_with_data_source
+from .input_fn import input_fn, pai_maxcompute_input_fn
+sns_colors = sns.color_palette('colorblind')
+# Disable Tensorflow INFO and WARNING logs
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+
+try:
+    import sqlflow_models
+except:
+    pass
+
+# TODO(shendiaomo): Remove after we fully upgrade to TF2.0
+TF_VERSION_2 = True
+TF_VERSION_PARTS = tf.__version__.split(".")
+if int(TF_VERSION_PARTS[0]) == 1:
+    TF_VERSION_2 = False
+
+# Disable Tensorflow INFO and WARNING logs
+if TF_VERSION_2:
+    import logging
+    tf.get_logger().setLevel(logging.ERROR)
+else:
+    tf.logging.set_verbosity(tf.logging.ERROR)
+    from .pai_distributed import define_tf_flags
+
+def analyze(datasource, estimator_cls, select, feature_columns, feature_column_names,
```

Comment:
In https://github.com/sql-machine-learning/sqlflow/pull/1587, I am renaming Analyze into Explain. How about we rename this function `explain`?

Reply:
I'll fix it in the next PR.